snprc_ehr/src/client/SndEventsWidget/components/EventListingGridPanel.tsx

```diff
@@ -145,23 +162,29 @@ export const EventListingGridPanelImpl: FC<EventListingProps> = memo((props: Eve
                 />
             )}
             {showDialog === 'edit' && (
-                <ProcedureEntryModal onCancel={closeDialog}
+                <ProcedureEntryModal onCancel={handleCloseUpdateModal}
```

Comment:
onCancel , we can directly closeDialog here Jay instead of calling the handleCloseUpdateModal and doing closeDialog in that handle function.

Reply:
just as you did in line 174